scripts/update-xkcd-entries.mjs

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line spaced-comment
+/// <reference lib="ESNext" />
+
+import { readFile, writeFile } from 'node:fs/promises';
+import { fetchEntry, fetchLatest, outputFile } from './shared/xkcd.mjs';
+
+const output = JSON.parse(await readFile(outputFile, 'utf8'));
+const lastRecordedId = output.at(-1).id;
+
+const last = await fetchLatest();
+if (last === null) process.exit(1);
+
+const lastId = last.id;
+if (lastId === lastRecordedId) {
+	console.log('No new entries to add.');
+	process.exit(0);
+}
+
+for (let i = lastRecordedId + 1; i < lastId; i++) {
+	const entry = await fetchEntry(i);
+	if (entry === null) continue;
+
+	output.push(entry);
+	process.stdout.write(`\rWritten ${i} out of ${lastId} (${Math.round((i / lastId) * 100)}%)`);
+}
+
+output.push(last);
+process.stdout.write(`\rWritten ${lastId} out of ${lastId} (100%)`);
+
+await writeFile(outputFile, JSON.stringify(output, undefined, '\t'), 'utf8');
```

Comment:
```suggestion
await writeFile(outputFile, JSON.stringify(output), 'utf8');
```

Do not apply formatting for a smaller file

Reply:
I applied formatting so I can curate and fix any formatting issue, take the following link:

https://xkcd.com/327/info.0.json

It has the substring `In a wayâ\nPhone`, when it should be `In a way —`. There are numerous formatting issues, and they're way too few to make an autofix, specially since xkcd hasn't done transcripts in a long while.

With the formatting, I can easily see new entries and fix them, I'll also retroactively fix those entries.